mycore-iview2/src/main/resources/xsl/mcr-module-startIview2.xsl

```diff
@@ -85,7 +85,7 @@
       </xsl:choose>
     </xsl:variable>
 
-    <img src="{concat($WebApplicationBaseURL,'servlets/MCRTileCombineServlet/',$imageSize,'/',$derivate,'/',$file)}" style="{$style}" class="{$class}" />
+    <img src="{concat($WebApplicationBaseURL,'servlets/MCRTileCombineServlet/',$imageSize,'/',$derivate,'/',encoder:encode($file))}" style="{$style}" class="{$class}" />
```

Comment:
do not use URLEncoder to encode URI path. URLEncoder should only be used for parameter values. Please use `org.mycore.common.xml.MCRXMLFunctions#encodeURIPath`